niworkflows/interfaces/utils.py

```diff
@@ -710,15 +710,45 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class DictMergeInputSpec(BaseInterfaceInputSpec):
+    in_dicts = traits.List(
+        traits.Either(traits.Dict, traits.Instance(OrderedDict)),
+        desc='Dictionaries to be merged. In the event of a collision, values '
+             'from dictionaries later in the list receive precedence.')
+
+
+class DictMergeOutputSpec(TraitedSpec):
+    out_dict = traits.Either(traits.Dict, traits.Instance(OrderedDict),
```

Comment:
As it is written right now, out_dict is always a dict.

```suggestion
    out_dict = traits.Dict(desc='Merged dictionary')
```